stylesheets/bootstrap-extensions.css

```diff
@@ -123,6 +128,14 @@ h1.panel-title {
     max-width: 350px;
 }
 
+.inline-screenshot.pull-right {
```

Comment:
I'll just trust you on these CSS changes for now :wink: We'll need to do a sweep of pages that used these styles before to make sure they still look good.